packages/react-reconciler/src/__tests__/ReactAsyncActions-test.js

```diff
@@ -822,6 +822,141 @@ describe('ReactAsyncActions', () => {
     );
   });
 
+  // @gate enableAsyncActions
+  test('useOptimistic rebases pending updates while dropping finished updates on top of passthrough value', async () => {
```

Comment:
It's currently not dropping the finished updates but should as far as I can tell.